docs/src/main/sphinx/release/release-469.md

```diff
@@ -0,0 +1,192 @@
+* Fix failures with recursive delete operations on S3Express preventing usage
+  successful usage for fault-tolerant execution. ({issue}`24763`)
+
+## Web UI
+
+* Add support for filtering queries by client tags. ({issue}`24494`)
+
+## JDBC driver
+
+* Add `planningTimeMillis`, `analysisTimeMillis`, `finishingTimeMillis`,
+  `physicalInputBytes`, `physicalWrittenBytes`, `internalNetworkInputBytes` and
+  `physicalInputTimeMillis` to `io.trino.jdbc.QueryStats`. ({issue}`24571`,
+  {issue}`24604`)
+* Add the `validateConnection` connection property to allow validating
+  connection, and improve the `Connection.isValid(int)` method so it validates
+  connection and credentials. ({issue}`24127`, {issue}`22684`)
+* Prevent failures when using the spooling protocol with a cluster using its own
+  certificate chain. ({issue}`24595`)
+* Fix deserialization failures with `SetDigest`, `BingTile`, and `Color` types. ({issue}`24612`)
+
+## CLI
+
+* Prevent failures when using the spooling protocol with a cluster using its own
+  certificate chain. ({issue}`24595`)
+* Fix deserialization of `SetDigest`, `BingTile`, and `Color` types. ({issue}`24612`)
+
+## BigQuery connector
+
+* Allow configuration of the channel pool for gRPC communication with BigQuery. ({issue}`24638`)
+
+## ClickHouse connector
+
+* {{breaking}} Raise minimum required versions to ClickHouse 24.3 and Altinity
+  22.3. ({issue}`24515`)
+* Fix failure when updating values to `NULL`. ({issue}`24204`)
+
+## Delta Lake connector
+
+* Add support for SSE-C in S3 security mapping. ({issue}`24566`)
+* Allow configuring the endpoint for the Google Storage file system with the
+  `gcs.endpoint` property. ({issue}`24626`)
+* Improve performance of reading from new Delta Lake table data by compressing
+  files with `ZSTD` by default, instead of the previous `SNAPPY`.
+  ({issue}`17426`)
+* Improve performance of queries on tables with large transaction log JSON
+  files. ({issue}`24491`)
+* Improve performance of reading from Parquet files with a large number of row
+  groups. ({issue}`24618`)
+* Improve performance for the `OPTIMIZE` statement by enabling concurrent
+  execution. ({issue}`16985`)
+* Improve performance of reading from large files on S3. ({issue}`24521`)
+* Correct catalog information in JMX metrics when using file system caching with
+  multiple catalogs. ({issue}`24510`)
+* Fix table read failures when using the Alluxio file system. ({issue}`23815`)
+* Fix incorrect results updating tables with deletion vectors enabled. ({issue}`24648`)
+* Fix incorrect results when reading from tables with deletion vectors enabled. ({issue}`22972`)
+
+## Elasticsearch connector
+
+* Improve performance of queries that reference nested fields from Elasticsearch
+  documents. ({issue}`23069`)
+
+## Faker connector
+
+* Add support for views. ({issue}`24242`)
+* Support generating sequences. ({issue}`24590`)
+* {{breaking}} Replace predicate pushdown with the `min`, `max`, and `options`
```

Comment:
This is not about replacing predicate pushdown, but changing the mechanism for specifying the constraints of the generated data. Previously, it would infer them from the WHERE clause. Now, they need to be provided as part of the table / column declaration via column properties.

Reply:
I got that wording from @nineinchnick .. will reword though

Reply:
Done